.cci.jenkinsfile

```diff
@@ -0,0 +1,43 @@
+// Documentation: https://github.com/coreos/coreos-ci/blob/main/README-upstream-ci.md
+
+properties([
+    // abort previous runs when a PR is updated to save resources
+    disableConcurrentBuilds(abortPrevious: true)
+])
+
+
+buildPod(runAsUser: 0, memory: "2Gi", cpu: "2") {
+      checkout scm
+      stage("Install packit") {
+        shwrap("""
+          dnf install packit -y
+        """)
+      }
+      stage("Build kdump RPM") {
+        // note: coreos upstream CI only run on x86 so we hardcode the architecture
+        shwrap("""
+          packit build locally --release-suffix coreos.tests
+          mv x86_64/kdump-utils*coreos.tests*.rpm ./kdump-coreos-ci.rpm
+        """)
+      }
+      stash includes: 'kdump-coreos-ci.rpm', name: 'kdump-rpm'
+}
+
+// Build FCOS with the kdump rpm
+cosaPod(runAsUser: 0, memory: "4Gi", cpu: "4") {
+
+   stage("Build FCOS") {
+       unstash 'kdump-rpm'
+       shwrap("""
+         coreos-assembler init --force https://github.com/coreos/fedora-coreos-config
+         mv kdump-coreos-ci.rpm overrides/rpm
+         cosa fetch --with-cosa-overrides
+         cosa build
+       """)
```

Comment:
This is fine, though honestly we should probably add a e.g. `rpms` argument to `cosaBuild` so all this becomes `cosaBuild(rpms: ['kdump-rpm'])`. Similar to the `overlays` dir.

Reply:
Nice suggestion. I don't know all the features of coreos-ci-lib (yet) :smiley: